.storybook/main.js

```diff
@@ -11,6 +11,20 @@ module.exports = {
     "@storybook/addon-links",
     "@storybook/addon-essentials"
   ],
+  babel: async (options) => {
+    return ({
+    ...options,
+    plugins: [
+      ...options.plugins,
+      [
+        "formatjs",
+        {
+          "idInterpolationPattern": "[sha512:contenthash:base64:6]",
+          "ast": true
+        }
```

Comment:
It turns out the `formatMessages` call is basically a placeholder, and a build-tool goes through, add's the ID & removes the unnecessary data from the built code.

I've added it to Storybook and all 3 sites